Cargo.toml

```diff
@@ -6,7 +6,8 @@ resolver = "2"
 [workspace.dependencies]
 anyhow = "1"
 chrono = "0.4"
-clap = "4.1"
+# this is latest version with small number of deps
+clap = "=4.1.14"
```

Comment:
I don't think that reducing the number of deps is so critical here that we would need to pin specific versions, that should be left to really rare circimstances. Just leave it at `"4"`.

Reply:
One of sources of dep number increase here https://github.com/rust-lang/rustc-perf/pull/2034#issuecomment-2607818469 was clap, so i tried to prevent it. Ok.